landcover.mss

```diff
@@ -546,13 +527,18 @@
 }
 
 #landuse-overlay {
-  [landuse = 'military'][zoom >= 10]::landuse {
+  [feature = 'military'][zoom >= 10] {
```

Comment:
I would prefer to have this as `feature = 'landuse_military'` to keep consistency with the other feature constructs.